### PR TITLE
Remove new address alert

### LINKED ISSUE #14783: Remove new address alert
### Description
Users are having issues with the multiple stacking alerts which make it hard to read transaction information. In particular the "new address detected" alert and "new gas experience alert"

**Solution:** Remove the "new address detected" as it is not dismissible and has tested poorly in user testing.

Relative quotes from [Slack thread](https://consensys.slack.com/archives/GTQAGKY5V/p1653506168360349)

> A user just reached out to me to share this video relating to an issue where multiple notifications on the confirmation screen make it impossible to read the details of a signature confirmation. The video he shared with me doesn't exactly show the issue he is describing though, but I think he wasn't sure how to recreate the issue he originally had. But it does show the stacking of notifications.

> ++ totally agree. I actually think we should get rid of this address found alert until we improve it. [@eddie.pearson](https://consensys.slack.com/team/U02EX8H722J) talked to users in research sessions and NONE of the users understood this alert or took action on it. We launched it without any testing which I think was a mistake and it clutters up the confirmation and may create more problems than good

Screen Shot 2022-05-25 at 1 32 38 PM

https://user-images.githubusercontent.com/8112138/170365787-3eba2884-0f83-492f-9288-7255163d7593.mov

### PR DESCRIPTION
**Pull request by ryanml**
Fixes: https://github.com/MetaMask/metamask-extension/issues/14783

## Explanation
See background in issue

## Manual Testing Steps
1. Go through the send flow attempting to send to an account that is not in your address book
2. Ensure the New address detected notification does not show (See videos in Issue)

### CODE REVIEW

**Review by georgewrmarshall — DISMISSED** [review]
Tested sending Rinkeby eth to new address. LGTM!

File: app/_locales/ar/messages.json [hunk 1]
```diff
@@ -586,9 +586,6 @@
   "newAccount": {
     "message": "حساب جديد"
   },
```
Comment (Nunnaphat1234) [inline app/_locales/ar/messages.json:588]: 7b478d7a0e56fb414f2b839e5f0007e518d800f9

File: app/_locales/ar/messages.json [hunk 2]
```diff
@@ -586,9 +586,6 @@
...
-  "newAccountDetectedDialogMessage": {
```
Comment (Nunnaphat1234) [inline app/_locales/ar/messages.json:589]: Hi key. Json walled

**Review by Nunnaphat1234 — COMMENTED** [review]
Hi

**Review by segun — DISMISSED** [review]
I tested this and I can confirm the Add contact dialog is no longer shown

**Comment by darkwing** [comment]
Tested and works well but I think we should add some padding here:
NoPadding

**Comment by darkwing** [comment]
Took the opportunity to update the padding:
PAddingFix

**Comment by Nunnaphat1234** [comment]
Hello oppa, phishing is overkill now. I can't read English very well. What will you do? now account NPM Someone came to create a stack. There is a picture. You need to help me. You can contact me by email.

**Comment by Nunnaphat1234** [comment]
Hello oppa, phishing is overkill now. I can't read English very well. What will you do? now account NPM Someone came to create a stack. There is a picture. You need to help me. You can contact me by email.

**Comment by Nunnaphat1234** [comment]
I allow the team Github Make a decision instead of doing it with autonomy in order to maintain the benefit. and enormous damage I don't know. I can't fight him anymore.

**Comment by Nunnaphat1234** [comment]
I allow the team Github Make a decision instead of doing it with autonomy in order to maintain the benefit. and enormous damage I don't know. I can't fight him anymore.

**Comment by georgewrmarshall** [comment]
Further riffing on this. To remove assumed padding and to clean up the jsx a bit

## Before

Screen Shot 2022-06-15 at 2 21 43 PMScreen Shot 2022-06-15 at 2 21 04 PM

## After

Screen Shot 2022-06-15 at 2 16 32 PMScreen Shot 2022-06-15 at 2 18 03 PM

**Review by brad-decker — DISMISSED** [review]

**Review by darkwing — DISMISSED** [review]

**Comment by bschorchit** [comment]
Noticed this PR got the two needed approvals, but ended up not being merged due to conflicts. Should we resolve them? cc: @ryanml

**Comment by brad-decker** [comment]
@ryanml can you give this some TLC so we can land it? It's P1

**Review by brad-decker — DISMISSED** [review]

**Review by georgewrmarshall — DISMISSED** [review]
LGTM

**Review by Nunnaphat1234 — COMMENTED** [review]
Help me to go abroad to buy tickets. Can a plane take me?

**Comment by brad-decker** [comment]
Converting to draft for now

**Comment by pedronfigueiredo** [comment]
I have read the CLA Document and I hereby sign the CLA

**Comment by pedronfigueiredo** [comment]
I ran into an interesting issue on `edit-gas-fee.spec.js`, coming from the `NumericInput` component inside `FormField`, inside `base-fee-input`.

Decimal separators `,` or `.` cannot be written at the end of the input field. Doing so causes the input field to clear.

Writing the `,` separator in the middle of the number of characters works.

Since this is unrelated to this branch's change (develop is similarly broken), I modified said broken e2e test case so that it wouldn't fail. The bug is captured on a separate issue (https://github.com/MetaMask/metamask-extension/issues/16703), as part of which it should be fixed and tested.

**Review by jpuri — APPROVED** [review]
Nice cleanup 👍

**Review by brad-decker — APPROVED** [review]

**Pull request merged**